rtv/page.py

```diff
@@ -388,6 +440,10 @@ def _edit_cursor(self, attribute=None):
         if self.nav.absolute_index < 0:
             return
 
+        # Don't allow the cursor to go over the subwindow number
```

Comment:
This check should be integrated into `move_page()`, not here. We should never allow an invalid cursor position.